unpackaged/config/trial_pmm/layouts/Contact-___NAMESPACE___Contact Layout.layout

```diff
@@ -0,0 +1,733 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm guessing this layout is intended to replace the layout that currently exists in the `unpackaged/config/trial/layouts` folder. We'll need to make sure to keep these in sync if we decide to make changes to the normal trial layout.

Reply:
That's accurate. I chose to keep them separate, at least for now, on the thesis that we may be asked for the ability to deploy a PMM-less trial configuration. I'd like to reconverge them if that proves not to be the case.